src/popup/components/ReverseAlias.vue

```diff
@@ -0,0 +1,136 @@
+<template>
+  <div class="content">
+    <div class="p-2 container">
+      <!-- Reverse-alias screen -->
+      <div class="m-2 p-2" v-if="!createdReverseAlias">
+        <p>
+          Create reverse-alias for
+          <span class="font-weight-bold">{{ alias.email }}</span>
+        </p>
+        <small>
+          To send an email from your alias to a contact, you need to create a
+          reverse-alias, a special email address. When you send an email to the
+          reverse-alias, the email will be sent from your alias to the
+          contact.<br /><br />
+          This Youtube video can also quickly walk you through the steps:
+          <a
+            href="https://www.youtube.com/watch?v=VsypF-DBaow"
+            target="_blank"
+            rel="noopener noreferrer"
+            >How to send emails from an alias</a
+          >
+        </small>
+        <br /><br />
+        <label>
+          Receiver:
+          <font-awesome-icon
+            v-b-tooltip.hover.top="'Where do you want to send the email?'"
+            icon="question-circle"
+          />
+        </label>
+        <b-input
```

Comment:
can we allow hitting enter will create  a reverse-alias here (without clicking on the button)?

Reply:
Added on https://github.com/simple-login/browser-extension/pull/104/commits/2d67d301fb32cf1a7e9318e98f8afe108c12339b